packages/esm-ugandaemr-app/src/pages/clinical-patient-summary/clinical-patient-summary-tabs/clinical-patient-summary-tabs.component.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Tab, TabList, TabPanel, TabPanels, Tabs } from '@carbon/react';
+import styles from './clinical-patient-summary-tabs.scss';
+import SubjectiveFindingsComponent from './subjective-findings.component';
+import ObjectiveFindingsComponent from './objective-findings.component';
+import TreatmentPlanComponent from './treatment-plan.component';
+import AssessmentComponent from './assessment.component';
+
+export const ClinicalPatientSummaryTabs = () => {
+  const { t } = useTranslation();
+  const [selectedTab, setSelectedTab] = useState(0);
+  return (
+    <div className={styles.tabcontainer}>
+      <Tabs selectedIndex={selectedTab} onChange={({ selectedIndex }) => setSelectedTab(selectedIndex)}>
```

Comment:
can we make this configurable. so that we can just add and remove these tabs without necessarily rewritting code and deleting somethings